### PR TITLE
Vox-7 Theming

### PR DESCRIPTION
**Pull request by marcotroisi**

### CODE REVIEW

File: src/Vox/Admin/Controller/BaseController.php [hunk 1]
```diff
@@ -65,4 +67,13 @@ public function logoutAction()
 
         return new ResponseRedirect($this->app->router()->make('admin'));
     }
+
+    public function michalAction()
```
Comment (potfur) [inline src/Vox/Admin/Controller/BaseController.php]: Temporary code?

File: src/Vox/Admin/Controller/PostController.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    /**
+     * Returns list of Posts
+     *
+     * @return Response
+     */
+    public function listAction()
+    {
+        return new Response(
```
Comment (potfur) [inline src/Vox/Admin/Controller/PostController.php:52]: Maybe create class `\Vox\Response\ResponseJSON` extending `Response` object that accepts array as content so we don't this each time.

File: src/Vox/Admin/View/index.twig [hunk 3]
```diff
@@ -1,43 +1,34 @@
-<!DOCTYPE html>
```
Comment (potfur) [inline src/Vox/Admin/View/index.twig:1]: Behat tests should be updated to work with new layout or preserve previous key elements.

File: src/Vox/Admin/View/Post/edit.twig [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{% extends "Vox:Admin:template" %}
```
Comment (potfur) [inline src/Vox/Admin/View/Post/edit.twig]: Maybe we should rename `template` to `layout` ?

File: src/Vox/Admin/View/Post/edit.twig [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+                <br><br>
+                <button type="submit" class="btn btn-primary btn-lg" ng-submit="savePost()">{% trans "Save" %}</button>
+
+            </div>
+            <div class="col-sm-2">
+                <!-- this column is to be used for more controls in the future -->
+            </div>
+        </div>
+
+        {% include 'Vox:Admin:footer' %}
```
Comment (potfur) [inline src/Vox/Admin/View/Post/edit.twig]: I think this should go to `template`

File: src/Vox/Core/CssManager.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @author Marco Troisi <hello@marcotroisi.com>
+ * @created 02.01.15
+ */
+
+namespace Vox\Core;
+
+
+class CssManager implements ResourceInterface
```
Comment (potfur) [inline src/Vox/Core/CssManager.php]: There is something to manage resources in `moss-twig-bridge` : https://github.com/potfur/moss-twig-bridge

**Comment by potfur** [comment]
Please don't merge this yet.

File: src/Vox/Admin/Controller/PostController.php (hunk 2 above)
Comment (marcotroisi) [inline src/Vox/Admin/Controller/PostController.php:52]: Yes, but it would make sense to have it directly in Moss framework, not here.

**Pull request merged**